restcomm/restcomm.mscontrol.mms/src/main/java/org/restcomm/connect/mscontrol/mms/MmsBridgeController.java

```diff
@@ -203,14 +204,25 @@ private void saveRecording() {
             builder.setAccountSid(accountId);
             builder.setCallSid(callId);
             builder.setDuration(duration);
-            builder.setApiVersion(runtimeSettings.getString("api-version"));
+            String apiVersion = runtimeSettings.getString("api-version");
```

Comment:
shall we replace hardcoded "api-version" with String constant that can be reused?